k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -138,17 +138,17 @@
     private static final String EXTRA_MESSAGE_REFERENCE = "message_reference";
 
     private static final String STATE_KEY_ATTACHMENTS =
-        "com.fsck.k9.activity.MessageCompose.attachments";
+            "com.fsck.k9.activity.MessageCompose.attachments";
```

Comment:
Is the whitespace changes from Android Studio having applied the settings JAR file? https://github.com/k9mail/k-9/wiki/CodeStyle